From 7a480b63d7e4c724cfe377672c9ad417eb33eceb Mon Sep 17 00:00:00 2001
From: Zhen Chen <chenzhen@rock-chips.com>
Date: Thu, 10 Jan 2019 10:13:21 +0800
Subject: [PATCH] arm64: configs: rockchip_linux_defconfig: add
 CONFIG_MALI_MIDGARD_FOR_LINUX back

In addition, enable CONFIG_MALI_MIDGARD and disable CONFIG_MALI_BIFROST
for rk3399.
Linux devices of rk3326 or px30 have their own defconfig files,
unlike Android devices.

Change-Id: I8bd8c10fad78f5a6a83d1fdd9e99850dda49ac47
Signed-off-by: Zhen Chen <chenzhen@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index b4ab4e2f0fbd..494c2f398f68 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -331,18 +331,14 @@ CONFIG_MALI450=y
 CONFIG_MALI_SHARED_INTERRUPTS=y
 CONFIG_MALI_DT=y
 CONFIG_MALI_DEVFREQ=y
-CONFIG_MALI_MIDGARD=m
+CONFIG_MALI_MIDGARD_FOR_LINUX=y
+CONFIG_MALI_MIDGARD=y
 CONFIG_MALI_EXPERT=y
 CONFIG_MALI_PLATFORM_THIRDPARTY=y
 CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI_DEBUG=y
 CONFIG_MALI_PWRSOFT_765=y
 CONFIG_MALI_BIFROST_FOR_LINUX=y
-CONFIG_MALI_BIFROST=y
-CONFIG_MALI_BIFROST_DEVFREQ=y
-CONFIG_MALI_PLATFORM_NAME="rk"
-CONFIG_MALI_BIFROST_EXPERT=y
-CONFIG_MALI_BIFROST_DEBUG=y
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-- 
2.35.3

